tokens/src/zIndex.json

```diff
@@ -0,0 +1,36 @@
+{
+  "z-index": {
+    "below-zero": {
+      "$description": "Stack an element behind elements that do not have a stacking order applied.",
+      "value": "-1"
+    },
+    "zero": {
+      "$description": "Stack an element behind elements that do not have a stacking order applied.",
```

Comment:
is it correct, that they both have the same description?

Reply:
no. 294edaf953a9fdede08d32d64eef92c7360bdfe5